src/glossary.rst

```diff
@@ -1247,7 +1251,11 @@ Glossary
          :term:`Tasks <task>` can be :term:`held <hold>` with ``cylc hold`` and
          :term:`released <release>` with ``cylc release``.
 
-         When a workflow is resumed, any held tasks remain held.
+         When a paused workflow is resumed, any held tasks remain held.
+
+         If a running task is held its running job is not affected, but it will
+         not submit any more jobs (such as automatic :term:`retries <retry>`)
+         until released. The children of held tasks are not automatically held.
```

Comment:
I get what you mean here (children = graph children), but I don't think users will.

We can omit this (there is no reason to think that this might happen):

```suggestion
         until released. 
```